File: 1170 - Blobs.py
import math

minFood = 1.0

def main():
    numberOfCases = int(input())

    for case in range(numberOfCases):
        foodAmount = float(input())
        daysUntilDeath = 0
        while foodAmount > minFood:
            foodAmount = foodAmount / 2
            daysUntilDeath += 1
        
        print(daysUntilDeath, "dias")

if __name__ == "__main__":
    main()

File: 1170 - Blobs.c
#include <stdio.h>

int main(){
    float minFood = 1.0, foodReceived;
    int numberOfCases, i;

    scanf("%d", &numberOfCases);

    for(i=0;i<numberOfCases;i++){

        int daysUntilDeath = 0;
        scanf("%f", &foodReceived);
        
        while(foodReceived > minFood){
            daysUntilDeath++;
            foodReceived /= 2;
        }

        printf("%d dias\n", daysUntilDeath);
    }
    return 0;
}